maps/_common/areas/station/storage.dm

```diff
@@ -53,3 +53,8 @@
 	requires_power = 0
 	name = "\improper Test Room"
 	icon_state = "storage"
+
+/area/storage/shields
+	name = "Station Shield Control"
```

Comment:
This should have \improper at the front of it.

Reply:
Got it